macros/fivetran/base/fivetran_fb_url_tag.sql

```diff
@@ -0,0 +1,81 @@
+{% macro fivetran_fb_url_tag() %}
+
+    {{adapter_macro('facebook_ads.fivetran_fb_url_tag')}}
+
+{% endmacro %}
+
+{% macro default__fivetran_fb_url_tag() %}
+
+with base as (
+    
+select distinct
```

Comment:
this is a bit unusual--just doing a straight distinct. what's going on here that recommends this approach to you?

Reply:
fivetran pulls in duplicates for no discernible reason. Those are the only 4 columns in the table that fivetran doesn't create so this is just to deduplicate the table. Is there a preferred method for doing this?

Reply:
moved distinct to the last step of the model